tools/engine_tool/lib/src/commands/test_command.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:engine_build_configs/engine_build_configs.dart';
+import 'package:path/path.dart' as p;
+
+import '../build_utils.dart';
+import '../gn_utils.dart';
+import '../proc_utils.dart';
+import '../worker_pool.dart';
+import 'command.dart';
+import 'flags.dart';
+
+/// The root 'test' command.
+final class TestCommand extends CommandBase {
+  /// Constructs the 'build' command.
+  TestCommand({
+    required super.environment,
+    required Map<String, BuilderConfig> configs,
+  }) {
+    builds = runnableBuilds(environment, configs);
+    debugCheckBuilds(builds);
+    argParser.addOption(
+      configFlag,
+      abbr: 'c',
+      defaultsTo: 'host_debug',
```

Comment:
Hey for these should we be defaulting to `host_debug_unopt`? I imagine unoptimized builds complete faster. I don't feel strongly either ways, just a thought.

Reply:
I think we should prefer the non-unopt debug build variants. cmiiw: unopt isn't materially faster to build and doesn't give us a better debugging experience.